easybuild/framework/easyconfig/default.py

```diff
@@ -158,6 +158,11 @@
 
     # MODULES easyconfig parameters
     'whatis': [None, "List of brief (one line) package description entries", MODULES],
+    'usage': [None, "Usage instructions for the software", MODULES],
+    'docpaths': [None, "List of paths of installed documentation relative to package root dir", MODULES],
+    'docurls': [None, "List of URLs to package documentation", MODULES],
+    'support': [None, "Package-specific support/bug report address", MODULES],
+    'contact': [None, "Site contact for the package", MODULES],
```

Comment:
Hmm... I just noticed that there is already a `docurls` parameter defined in the `MANDATORY` section, though it is not (yet) enforced.  I guess it hasn't been used so far?

Regarding the "messiness": Would it make sense to add a `DOCUMENTATION` group and add all those variables to it?  Though `description` would then need to be in `MANDATORY` and `DOCUMENTATION`...